setup.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) IBM Corporation 2018
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
What is this license? And year? ;)

In README you (or IBM github admin) have clearly specified how the license header should look like... ;)

Reply:
Fixed by rebasing all the commits!
The IBM GitHub admin proposes that SPDX format in `CONTRIBUTING.md`.